config/shipwright/build/strategy/buildah.yaml

```
@@ -168,6 +168,9 @@ spec:
         - $(params.registries-insecure[*])
         - --registries-search
         - $(params.registries-search[*])
+      volumeMounts:
+      - mountPath: /etc/pki/entitlement
+        name: rhel-entitlement
```

nit: name change to align with what we have in the BuildConfig docs

```suggestion
        name: etc-pki-entitlement
```